app/controllers/users/omniauth_controller.rb

```diff
@@ -6,10 +6,15 @@ def callback
 
       if @user
         sign_in @user
+        set_id_token
         redirect_to session[:requested_url]
       else
         redirect_to users_none_url
       end
     end
+
+    def set_id_token
```

Comment:
maybe call it `store_id_token` ? `set` makes me think of an instance variable setter